CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # honeycomb-opentelemetry-python changelog
 
+## [0.1.0b0] - 2023-03-29
+
+Initial beta release of Honeycomb's OpenTelemetry distribution for Python!
+
+### Maintenance
+
+- ci: require smoke tests for publish steps (#107) | [@pkanal](https://github.com/pkanal)
+- maint: drop poetry locks from example apps (#111) | [@JamieDanielson](https://github.com/JamieDanielson)
```

Comment:
Ride the Lightning!

![](https://media.tenor.com/PYFq3xQUeOgAAAAC/hangover-alan.gif)